iis-core/src/test/java/eu/dnetlib/iis/core/IntegrationTestPropertiesReader.java

```diff
@@ -97,4 +127,27 @@ private Properties readUserProperties() {
 		return userProperties;
 	}
 	
+	private String resolveUserPropertiesFilePath() {
+		String userPropertiesPath = System.getProperty("connectionProperties");
+		if (userPropertiesPath == null) {
+			return DEFAULT_USER_PROPERTIES_PATH;
+		}
+		return userPropertiesPath;
+	}
+	
+	private void createTempPropertiesFile() {
```

Comment:
I think it should be in a separate service method that takes Properties and file path as parameters.